lib/SettingsParser.d.ts

```diff
@@ -0,0 +1,6 @@
+import { FormattingSettingsSlice } from "./SettingsInterfaces";
+import powerbi from "powerbi-visuals-api";
```

Comment:
which version is required? is this relevant?

Reply:
version for what?

Reply:
of the api

Reply:
I added powerbi-visuals-api version 5.0.0 but i still wait for its PR